src/data-generator/declare.ts

```diff
@@ -34,11 +22,28 @@ import {
   FETCH_REGISTRATION_QUERY,
   SEARCH_EVENTS
 } from './queries'
-import { join } from 'path'
-import { readFileSync } from 'fs'
+import {
+  BIRTH_ATTACHMENTS,
+  DEATH_ATTACHMENTS,
+  education,
+  identity,
+  maritalStatus,
+  location as locationTypes,
+  address,
+  informant,
+  mannerOfDeath,
+  birth,
+  attendant,
+  causeOfDeathMethod
+} from './options'
 
 const HOME_BIRTH_WEIGHT = 0.2
 const HOME_DEATH_WEIGHT = 0.2
+/*
+ * Number of attachments per record. Please see `BIRTH_ATTACHMENTS`
+ * and `DEATH_ATTACHMENTS`, this number cannot exceed the biggest
+ * index of either of them.
+ */
 const NUMBER_OF_ATTACHMENTS_PER_RECORD = process.env
   .NUMBER_OF_ATTACHMENTS_PER_RECORD
   ? parseInt(process.env.NUMBER_OF_ATTACHMENTS_PER_RECORD, 10)
```

Comment:
kinda forgot to move this myself, but should we move this to `data-generator/constants.ts` ? I think this is the only process.env that isn't being read from that file

Reply:
sounds good. I'll move it there

Reply:
or it can stay there I guess, because some of them are also in the index.ts file. Perhaps we can move all of them sometime in the future